src/main/python/pybuilder/plugins/python/coverage_plugin.py

```diff
@@ -44,45 +44,45 @@ def init_coverage_properties(project):
     project.set_property_if_unset("coverage_fork", False)
 
 
-def start_coverage(coverage):
-    coverage.erase()
-    coverage.start()
-
-
-def stop_coverage(coverage, project, logger):
-    reimport_source_modules(project, logger)
```

Comment:
ugh, except the pybuilder's own build doesn't run with module reload right after the coverage started. investigating